### ISSUE TITLE
Dockerfile does not build

### ISSUE THREAD

**Issue by peti** [haskell/hackage-server#835]
While building the docker container,  the installation of `postfix` requires user interaction that does not work:

~~~
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Postfix Configuration
---------------------

Please select the mail server configuration type that best meets your needs.

 No configuration:
  Should be chosen to leave the current configuration unchanged.
 Internet site:
  Mail is sent and received directly using SMTP.
 Internet with smarthost:
  Mail is received directly using SMTP or by running a utility such
  as fetchmail. Outgoing mail is sent using a smarthost.
 Satellite system:
  All mail is sent to another machine, called a 'smarthost', for delivery.
 Local only:
  The only delivered mail is the mail for local users. There is no network.

  1. No configuration  3. Internet with smarthost  5. Local only
  2. Internet Site     4. Satellite system
General type of mail configuration: 
~~~

There is no way to answer that question; user input in the shell has no effect.

**Comment by peti** [comment]
https://github.com/haskell/hackage-server/pull/826 addresses this issue, apparently, but the PR seems to be stuck.

**Comment by hvr** [comment]
It's fixed properly now via 3a5bc1a4b8e519cbf21bbb3f9f53a064bab42b37
